source/develop/release-management/features/storage/pass_discard_from_guest_to_underlying_storage.html.md

```diff
@@ -0,0 +1,234 @@
+---
+authors: ishaby
```

Comment:
You need to add an entry in `authors.yml` so this is translated to your full name.